pocs/state/states/default/pointing.py

```diff
@@ -49,6 +55,7 @@ def on_enter(event_data):
                     except Exception as e:
                         pocs.logger.error("Problem waiting for images: {}".format(e))
 
+            # Wait for images to complete
```

Comment:
We definitely need to put together a helper for this kind of loop, shared with observing.py (or whatever the name is... the one I recently changed).